.vscode/launch.json

```diff
@@ -5,7 +5,8 @@
       "type": "node",
       "request": "attach",
       "name": "Attach by Process ID",
-      "processId": "${command:PickProcess}"
+      "processId": "${command:PickProcess}",
+      "continueOnAttach": true
```

Comment:
What is this doing?

Reply:
Currently when you attach the debugger in vscode, you need to manually hit continue, this setting removes that step.

Reply:
seems nice!